src/main/telemetry/smartport.c

```diff
@@ -696,6 +683,66 @@ void processSmartPortTelemetry(smartPortPayload_t *payload, volatile bool *clear
                 }
                 break;
 #endif
+#if defined(USE_ESC_SENSOR_TELEMETRY) || defined(USE_DSHOT_TELEMETRY)
+            case FSSP_DATAID_RPM        :
+#ifdef USE_ESC_SENSOR_TELEMETRY
+                escData = getEscSensorData(ESC_SENSOR_COMBINED);
+                if (escData != NULL) {
+                    smartPortSendPackage(id, calcEscRpm(escData->rpm));
+                    *clearToSend = false;
+                }
+#endif // USE_ESC_SENSOR_TELEMETRY
+#ifdef USE_DSHOT_TELEMETRY
+#ifdef USE_ESC_SENSOR_TELEMETRY
+                if (useDshotTelemetry && !escData) {
+#else
+                if (useDshotTelemetry) {
+#endif // USE_ESC_SENSOR_TELEMETRY
+                    const unsigned nMotors = getMotorCount();
+                    if (nMotors > 0) {
+                        float eRpmSum = 0;
+                        for (unsigned i = 0; i < nMotors; i++) {
```

Comment:
```suggestion
                        for (uint8_t = 0; i < nMotors; i++) {
```

@daleckystepan does this matter in code size? Should we update other instances using for loops with values below 256?